gtsfm/data_association/point3d_initializer.py

```diff
@@ -193,9 +193,9 @@ def execute_ransac_variant(self, track_2d: SfmTrack2d) -> np.ndarray:
                 )
             except RuntimeError:
                 # TODO: handle cheirality exception properly?
-                logger.debug(
```

Comment:
I think we may never return the TODO above if we comment this out :)

Reply:
Yes true. But what is the appropriate way to handle it. We are running into so many cheirality exception and the terminal is full of this logging.

Reply:
Let's keep for now, I'm definitely motivated by the logging to try and fix it.